dev/io.openliberty.microprofile.faulttolerance.4.1.telemetry.metrics.2.0/bnd.bnd

```diff
@@ -0,0 +1,59 @@
+#*******************************************************************************
+# Copyright (c) 2018, 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+openTelemetryVersion=1.35.0
+
+Bundle-Name: io.openliberty.microprofile.faulttolerance.4.1.telemetry.metrics.2.0
+Bundle-SymbolicName: io.openliberty.microprofile.faulttolerance.4.1.telemetry.metrics.2.0
+
+Bundle-Description:MicroProfile Fault Tolerance telemetry integration
+
+WS-TraceGroup: FAULTTOLERANCE
+
+-dsannotations-inherit: true
+-dsannotations: io.openliberty.microprofile.faulttolerance.telemetry.metrics_20.MetricRecorderProviderImpl
+
+#We need an explicit version for mp config because we're not transforming this bundle.
+Import-Package: org.eclipse.microprofile.config;version="[3.0,4)", \
+        *
```

Comment:
Explicit version of config shouldn't be needed if you're depending on the correct version in the buildpath I think.